storage/migrations/08_evm_tokens_name_index.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add a trigram index for evm_tokens token names for search.
+
+BEGIN;
+
+CREATE EXTENSION pg_trgm;
+
+CREATE INDEX ix_evm_tokens ON chain.evm_tokens USING GIST (token_name gist_trgm_ops);
```

Comment:
https://www.postgresql.org/docs/current/pgtrgm.html

can someone check if this is right

Reply:
oh the index name was not good. fixed to be more descriptive